todo.go

```diff
@@ -241,15 +248,17 @@ type taskImpl struct {
 	text               string
 	priority           Priority
 	created, completed time.Time
+	attributes         map[string]string
```

Comment:
I don't see any real reason for this to be here? `comment` could just be a string field.